packages/@aws-cdk/aws-lambda-event-sources/lib/kafka.ts

```diff
@@ -213,6 +217,9 @@ export class SelfManagedKafkaEventSource extends StreamEventSource {
       case AuthenticationMethod.BASIC_AUTH:
         authType = lambda.SourceAccessConfigurationType.BASIC_AUTH;
         break;
+      case AuthenticationMethod.CLIENT_CERTIFICATE_TLS_AUTH:
+        authType = lambda.SourceAccessConfigurationType.of(AuthenticationMethod.CLIENT_CERTIFICATE_TLS_AUTH);
```

Comment:
can you add this as a static method to `lambda.SourceAccessConfigurationType` also? Not sure why there's this duplicate between `AuthenticationMethod` and `lambda.SourceAccessConfigurationType` but there's nothing we can do about it now.

Reply:
@kaizen3031593 I have updated the PR with the new static method. Thank you for reviewing it 💪